File: src/main/java/com/zippeykeys/praisebe/util/NetworkUtil.java
package com.zippeykeys.praisebe.util;

import java.util.UUID;

import com.zippeykeys.praisebe.Reference;
import com.zippeykeys.praisebe.data.DeityRegard;
import com.zippeykeys.praisebe.data.PBWorldData;

import lombok.experimental.UtilityClass;
import net.minecraft.entity.player.EntityPlayer;

@UtilityClass
public class NetworkUtil {
    private static PBWorldData worldData = Reference.INSTANCE.worldData();

    public static DeityRegard getRegard(EntityPlayer player) {
        return worldData.getDeityRegard(player);
    }

    public static DeityRegard getRegard(UUID uuid) {
        return worldData.getDeityRegard(uuid);
    }
}

File: src/main/kotlin/com/zippeykeys/praisebe/util/NetworkUtil.kt
package com.zippeykeys.praisebe.util

import com.zippeykeys.praisebe.Reference
import com.zippeykeys.praisebe.data.DeityRegard
import net.minecraft.entity.player.EntityPlayer
import java.util.UUID


private val worldData = Reference.INSTANCE.worldData()

fun getRegard(player: EntityPlayer): DeityRegard {
    return worldData.getDeityRegard(player)
}

fun getRegard(uuid: UUID): DeityRegard {
    return worldData.getDeityRegard(uuid)
}